src/main/java/com/iota/iri/model/ObsoleteTag.java

```diff
@@ -0,0 +1,12 @@
+package com.iota.iri.model;
+
+public class ObsoleteTag extends Tag {
+
+    public ObsoleteTag(Hash hash) {
+        super(hash);
+    }
+
+    public ObsoleteTag() {
```

Comment:
do we need this constructor?

Reply:
yes,
it throws `java.lang.InstantiationException: com.iota.iri.model.ObsoleteTag` if not.

Reply:
perhaps a comment like `// used by the persistence layer to instantiate the object` is in place.
We will add that similar comment to the other `Persistable` objects in another chance.